src/styles/card.scss

```diff
@@ -0,0 +1,112 @@
+.card {
+  width: 166px;
+  height: 358px;
+  background-color: #fff;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  line-height: 18px;
+  color: $main-text-color;
+  padding: 32px 16px 16px;
+
+  display: flex;
+  flex-direction: column;
+
+  cursor: pointer;
+  transition: transform 0.3s;
+
+  &:hover {
+    transform: scale(1.2);
+    color: #34568b;
```

Comment:
It's not clear why you're changing the color of the entire card on hover. It might be better to change only the elements that need to be highlighted. For example, the card's border or the text color of certain elements.